docs/image/index.rst

```diff
@@ -12,25 +12,39 @@ Introduction
 ============
 
 `gammapy.image` contains data classes and methods for image based analysis
-of gamma-ray data. Currently it includes multi purpose image processing
-methods as well. The goal longterm goal is to contribute most of these methods
-to `scipy.ndimage`, `scikit-image`_, `astropy`_ or `photutils`_, and to only keep
-gamma-ray analysis specific functionality here.
+of gamma-ray data.
+
 
 Getting Started
 ===============
 
-Most of the functions in this module have objects of type `numpy.array`
-or an `astropy.io.fits.ImageHDU` or `astropy.io.fits.PrimaryHDU`
-as input and output:
+The central data structure in `gammapy.image` is the `gammapy.image.SkyImage`
+class, which combines the raw data with WCS information, FITS I/O functionality
+and many other methods, that allow easy handling, processing and plotting of
+image based data. Here is a first example:
 
 .. code-block:: python
 
-   >>> from gammapy.datasets import poisson_stats_image
-   >>> from gammapy.image import lookup
-   >>> image = poisson_stats_image() # image is a 2D numpy array
-   >>> lookup(image, 42, 44, world=False)
-   3.0
+   >>> from gammapy.datasets import gammapy_extra
+   >>> from gammapy.image import SkyImage
+   >>> filename = gammapy_extra.filename('datasets/fermi_2fhl/fermi_2fhl_vela.fits.gz')
+   >>> skyimage = SkyImage.read(filename, ext=2)
+   >>> skyimage.show()
+
+This loads a prepared Fermi 2FHL FITS image of the Vela region, creates a
+`gammapy.image.SkyImage` and shows it on the the screen by calling `gammapy.image.SkyImage.show()`.
+The image looks as following:
+
+.. figure:: fermi_2fhl_vela.png
+
+   Smoothed counts map of the Vela region computed with gammapy (see `script
+   in gammapy-extra <https://github.com/gammapy/gammapy-extra/blob/master/datasets/fermi_2fhl/fermi_skyimages.py>`_)
+
+
+To explore further the SkyImage class try tab completion on the ``skyimage`` object
+in an interactive python environment or see the _`skymaps` page.
```

Comment:
Does this link work?
I don't think I've seen an underscore before the reference for a Sphinx ref!?